src/transaction-runner.coffee

```diff
@@ -117,32 +157,22 @@ class TransactionRunner
 
     # sandboxed mode - hook is a string - only sync API
     if typeof(hook) == 'string'
-      wrappedCode = """
-      // run the hook
-      var _func = #{hook};
-      _func(_data);
-
-      // setup the return object
-      var output = {};
-      output["data"] = _data;
-      output["stash"] = stash;
-      output;
-      """
+      wrappedCode = @sandboxedWrappedCode hook
 
       sandbox = new Pitboss(wrappedCode, {
         timeout: 500
       })
 
-      sandbox.run {context: {"_data": data, stash: @hookStash}, libraries: ['console']}, (err, result = {}) =>
+      sandbox.run
+        context:
+          "_data": data
+          stash: @hookStash
+          logs: []
```

Comment:
This should be imho `_` prefixed as well. It's kind of a private variable, which we don't want to let users manipulate.